draft-wussler-openpgp-pqc.md

```diff
@@ -735,14 +659,14 @@ The various procedures to perform the operations of an ECC-based KEM are
 defined in the following subsections. Specifically, each of these subsections
 defines the instances of the following operations:
 
-    (eccCipherText, eccKeyShare) <- eccKem.encap(eccPublicKey)
+    (eccCipherText, eccKeyShare) <- ECC-KEM.Encaps(eccPublicKey)
 
 and
 
-    (eccKeyShare) <- eccKem.decap(eccPrivateKey, eccCipherText)
+    (eccKeyShare) <- ECC-KEM.Decaps(eccPrivateKey, eccCipherText)
 
-The placeholder `eccKem` has to be replaced with the specific ECC-KEM from the
-row "ECC-KEM" of {{tab-ecdh-cfrg-artifacts}}, {{tab-ecdh-nist-artifacts}}, and
+To instantiate ECC-KEM, one must select a parameter set from
```

Comment:
```suggestion
To instantiate `ECC-KEM`, one must select a parameter set from
```

Reply:
Applied inline code marking also to ML-KEM in the anlagous sentence. See https://github.com/openpgp-pqc/draft-openpgp-pqc/pull/60/commits/fa1b1284d6bea9ceb1dafbbf470305744c0f0fe7